src/NServiceBus.RavenDB/Timeouts/TimeoutPersister.cs

```diff
@@ -174,7 +164,7 @@ IRavenQueryable<Timeout> GetChunkQuery(IDocumentSession session)
                         t.Id,
                         t.Time
                     })
-                    .Take(maximumPageSize)
+                    .Take(1024) // RavenDB server default maximum page size
```

Comment:
This is the cleanup chunk query. If you don't take 1024, you'll get client default 128.